### PR TITLE
Issues #194 - Clean up Wolfram syntax

### PR DESCRIPTION
**Pull request by ethanaward**
This partially addresses #194. The WA skill will now no longer search again for things it thought you meant, and instead will just say a possible alternative. Also, I've created some regex to parse definitions so that it will say only one definition of a word rather than all of the definitions it can find, as this often takes far longer than needed.

### CODE REVIEW

**Comment by Wolfgange3311999** [comment]
I think that `Did you mean` insinuates a response of which is not yet support. Could we change that to something like `Perhaps you meant`?

File: mycroft/skills/wolfram_alpha/__init__.py [hunk 1]
```diff
@@ -91,6 +91,7 @@ def query(self, query):
         response = requests.get(url, headers=headers)
         if response.status_code == 401:
             raise CerberusAccessDenied()
+        # logger.debug(response.content)
```
Comment (Wolfgange3311999) [inline mycroft/skills/wolfram_alpha/__init__.py]: Debug line accidentally added.

**Comment by Wolfgange3311999** [comment]
LGTM.

**Comment by isaacnward** [comment]
LGTM too, merging.

**Pull request merged**